analysis/parser/RawTextParser/src/main/kotlin/de/maibornwolff/codecharta/parser/rawtextparser/metrics/Metric.kt

```diff
@@ -0,0 +1,18 @@
+package de.maibornwolff.codecharta.parser.rawtextparser.metrics
+
+import de.maibornwolff.codecharta.parser.rawtextparser.model.FileMetrics
+
+interface Metric {
+
+    val name: String
+    val description: String
+
```

Comment:
Why doesn't the interface declare `value` as well here?

Reply:
Why should it be? of course you could do declare `value` and then override the get() method for each metric. But I don't think that this would make it easier as the value property may (as in the case at hand) never hold a value and just contain the current getValue() function as an overridden getter.

(Conceptually you should not write trivial getters/setters in Kotlin and instead refer to the property value directly. However, getValue() does not simply access the state, but instead actually performs an action (e.g. aggregation of data to a value), therefore I implemented it as a function. If you don't like getValue() we may change it to computeValue()/retrieveValue())